addons/mail/static/src/core/web/thread_actions.js

```diff
@@ -13,7 +13,7 @@ threadActionsRegistry
             return component.thread.isEmpty;
         },
         open(component) {
-            component.orm.silent.call("mail.message", "mark_all_as_read");
+            component.thread.markAllMessagesAsRead();
```

Comment:
This is not the same, `mark_all_as_read` (no param) is inbox messages.
thread is specific (for real thread, it doesn't work for inbox).

Reply:
The `orm` service is simply still used, this PR doesn't really make sense.

Reply:
The PR can define `useService("orm")` in this thread action, so the locality of concern is better.
This is not really worth a FIX in stable IMO, but whatever ;)